src/helm/benchmark/metrics/bhasa_metrics.py

```diff
@@ -0,0 +1,316 @@
+            Li, Yuan-Fang  and
+            Kang, Yong-Bin  and
+            Rahman, M. Sohel  and
+            Shahriyar, Rifat",
+        editor = "Zong, Chengqing  and
+            Xia, Fei  and
+            Li, Wenjie  and
+            Navigli, Roberto",
+        booktitle = "Findings of the Association for Computational Linguistics: ACL-IJCNLP 2021",
+        month = aug,
+        year = "2021",
+        address = "Online",
+        publisher = "Association for Computational Linguistics",
+        url = "https://aclanthology.org/2021.findings-acl.413",
+        doi = "10.18653/v1/2021.findings-acl.413",
+        pages = "4693--4703",
+        github = "https://github.com/csebuetnlp/xl-sum",
+    }
+    
+    """
+
+    def __init__(self, language: str = 'en'):
+        self.language: str = language
+        self.rouge_metrics = {
+            "rougeL": "xlsum_rouge_l",
+        }
+        self.rouge_scorer = self._get_bhasa_rouge_scorer(self.rouge_metrics)
+
+    def _get_bhasa_rouge_scorer(self, rouge_metrics: str) -> Callable[[str, str], float]:
+        language = "thai" if self.language == "th" else None
+        return rouge_scorer.RougeScorer(list(rouge_metrics.keys()), use_stemmer=False, lang=language)
+
+    def _compute_rouge(self, refs: List[str], pred: str) -> Dict[str, float]:
+        metrics: Dict[str, float] = {}
+
+        aggregator = BootstrapAggregator()
+        for ref in refs:
+            aggregator.add_scores(self.rouge_scorer.score(ref, pred))
+        aggregates = aggregator.aggregate()
+        for key, value in self.rouge_metrics.items():
+            metrics[value] = aggregates[key].mid.fmeasure * 100
+        return metrics
+
+    def _remove_braces(self, text: str) -> str:
+        if text.startswith("{"):
+            text = text[1:]
+        if text.endswith("}"):
+            text = text[:-1]
+        return text
+    
+    def evaluate(
+        self, scenario_state: ScenarioState, metric_service: MetricService, eval_cache_path: str, parallelism: int
+    ) -> MetricResult:
+        return super().evaluate(scenario_state, metric_service, eval_cache_path, parallelism=parallelism)
+
+    def evaluate_generation(
+        self,
+        adapter_spec: AdapterSpec,
+        request_state: RequestState,
+        metric_service: MetricService,
+        eval_cache_path: str,
+    ) -> List[Stat]:
+        refs: List[str] = [self._remove_braces(ref.output.text) for ref in request_state.instance.references]
+        inp: str = self._remove_braces(request_state.instance.input.text)
+
+        assert request_state.result is not None
+        pred: str = self._remove_braces(request_state.result.completions[0].text.strip())
+
+        result: List[Stat] = []
+
+        # Compute rouge metrics
+        result.extend([Stat(MetricName(name)).add(float(val)) for name, val in self._compute_rouge(refs, pred).items()])
+
+        return result
+
+def get_bhasa_machine_translation_metric_specs() -> List[MetricSpec]:
```

Comment:
Move these to a different file `bhasa_metric_specs.py` (follow the conventions in `common_metric_specs.py`).

The reason for doing this is that we don't want the `bhasa_run_specs` file to import` bhasa_metrics`, which transitively imports optional dependencies, otherwise this would cause `helm-run` to fail for someone who doesn't have the optional dependencies installed.